src/js/utils/DOMUtils.js

```diff
@@ -28,21 +29,19 @@ var DOMUtils = {
 
     var computeInnerBound = function (acc, key) {
       var val = parseInt(compstyle[key], 10);
-      if (_.isNaN(val)) {
+      if (isNaN(val)) {
```

Comment:
oh man I just found out about this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN